EDF.py

```diff
@@ -308,74 +427,84 @@ def readBlock(self, block):
-                raw = np.asarray(unpack('<{}h'.format(chan_info['n_samps'][i]), buf), dtype=np.float32)
+                
+                raw = np.asarray(
+                    unpack('<{}h'.format(
+                        chan_info['n_samps'][i]), buf), dtype=np.float32)
+                
                 raw *= self.calibrate[i]
                 raw += self.offset[i]  # FIXME I am not sure about the order of calibrate and offset
                 data.append(raw)
         return data
 
-    def readSamples(self, channel, begsample, endsample):
+    def read_samples(self, channel, begsample, endsample):
         meas_info = self.meas_info
         chan_info = self.chan_info
         n_samps = chan_info['n_samps'][channel]
         begblock = int(floor((begsample) / n_samps))
         endblock = int(floor((endsample) / n_samps))
-        data = self.readBlock(begblock)[channel]
+        data = self.read_block(begblock)[channel]
         for block in range(begblock+1, endblock+1):
-            data = np.append(data, self.readBlock(block)[channel])
+            data = np.append(data, self.read_block(block)[channel])
         begsample -= begblock*n_samps
         endsample -= begblock*n_samps
-        return data[begsample:(endsample+1)]
+        return data[begsample:(endsample + 1)]
 
-####################################################################################################
-# the following are a number  of helper functions to make the behaviour of this EDFReader
-# class more similar to https://bitbucket.org/cleemesser/python-edf/
-####################################################################################################
-
-    def getSignalTextLabels(self):
-        # convert from unicode to string
+    def get_signal_text_labels(self):
         return [str(x) for x in self.chan_info['ch_names']]
 
-    def getNSignals(self):
+    def get_n_signals(self):
         return self.meas_info['nchan']
 
-    def getSignalFreqs(self):
+    def get_signal_freqs(self):
         return self.chan_info['n_samps'] / self.meas_info['record_length']
 
-    def getNSamples(self):
+    def get_n_samples(self):
         return self.chan_info['n_samps'] * self.meas_info['n_records']
 
-    def readSignal(self, chanindx):
-        begsample = 0;
-        endsample = self.chan_info['n_samps'][chanindx] * self.meas_info['n_records'] - 1;
-        return self.readSamples(chanindx, begsample, endsample)
-
-####################################################################################################
-
-if False:
-    file_in = EDFReader()
-    file_in.open('/Users/roboos/day 01[10.03].edf')
-    print file_in.readSamples(0, 0, 0)
-    print file_in.readSamples(0, 0, 128)
-
-
-if False:
+    def read_signal(self, chanindx):
+        begsample = 0
+        endsample = (
+            self.chan_info['n_samps'][chanindx] * self.meas_info['n_records']
+            ) - 1
+        return self.read_samples(chanindx, begsample, endsample)
```

Comment:
```suggestion
    def read_signal(self, chidx):
        beg = 0
        end = self.chan_info['n_samps'][chidx] * self.meas_info['n_records']
        return self.read_samples(chidx, beg, end - 1)
```

Reply:
won't removing the sample word from the variables make it less dev-friendly? Although the variables will be defined properly in the documentation.

Reply:
It's a tradeoff with line breaks. Logical lines spread over several physical lines are hard to read. `beg` and `end` indicate beginning and end, while the samples bit is clear from the function call `read_samples`.